fabric-ca-client/test/FabricCAServices.js

```diff
@@ -249,7 +249,7 @@ describe('FabricCAServices', () => {
 
 		it('should reject on enroll failure', async () => {
 
-			getSubjectCommonNameStub.returns('mr_penguin');
+			getSubjectStub.returns('mr_penguin');
```

Comment:
The return value should probably be `/CN=mr_penguin` to make it a valid subject string. I know it doesn't break the code today but might confuse somebody in the future if they change the code do something with the subject and all these tests start failing.